ports/llhttp/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO nodejs/llhttp
+    REF refs/tags/release/v8.1.0
+    SHA512 3b79555f734a6a200a790f04f10be6d3ce5bd999c027a43c0900bb408e7d64be38fe4bbe9f15e57e389f07ba26b8089fc6ba5eef3b497742484f0719e92e9722
+)
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" LLHTTP_BUILD_STATIC)
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" LLHTTP_BUILD_SHARED)
+
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS

```

```suggestion
    DISABLE_PARALLEL_CONFIGURE
    OPTIONS
```
```
CMake Error at CMakeLists.txt:41 (configure_file):
  No error
```
https://github.com/nodejs/llhttp/blob/85dd446419966cafb5a7fdd5031e3113a0768e65/CMakeLists.txt#L41-L45
Since the upstream write new file to source path, in order to avoid this cmake error message, we should add this option to disable the source changes detection.